nextflow.config

```diff
@@ -188,6 +188,8 @@ params {
 
     arg_hamronization_summarizeformat                = 'tsv'
 
+    arg_skip_argnorm                                 = false
```

Comment:
ARGnorm is basically taking the hamronization summary and annotating it with ARO IDs, why not have it automatically run when Abricate, Amrfinderplus and deeprag  are activated? Im afraid this might create a confusion for the user, when they for example want to run RGI and fargene only and then activate argnorm, what will happen in that case? Is there a log warning for this case ?

Reply:
To reduce the number of very small jobs of something someone might not be interested in. 

It's run by default so I think it's fine to have skipping opt in

Reply:
So if someone only runs RGI, for example, will the pipeline just skip running ARGnorm by itself? Or do we have to explicitly add `--arg_skip_argnorm` to skip it?

Reply:
`argNorm` won't be run in that case, it only runs if `abricate`, `deeparg` or `amrfinderplus` are used.